snap/hooks/configure

```diff
@@ -52,6 +61,28 @@ handle_svc () {
     esac
 }
 
+restart_db_services () {
+    # since the configuration files have been modified, those
+    # changes need to be progpogated into relevant services, so
+    # restart the services with updated configuration if they were
+    # already running
+    for svc in ${DB_SERVICES[*]}; do
```

Comment:
I have a slight preference for helper functions to take arguments rather than use global variables but this is fine if you would rather change this later

Reply:
I just followed your lead (e.g. `ALL_SERVICES`) on this one. Let's leave this one alone.

Reply:
ah right I had missed that this variable was just globally assigned once, I thought it was mutated in the other functions, nvm me this is fine